CONTRIBUTING.md

```diff
@@ -110,7 +110,9 @@ Once you are ready to publish a release, branch off from `dev`
     ```
 For meaning of X, Y and Z version numbers, please refer to this [semantic versioning guidelines](https://semver.org/spec/v2.0.0.html).
 
-In this branch, you should normally only update the version number in the `CHANGELOG.md` and `setup.py` files.
+In this branch, you should normally only update the version number in the 
+`src/mvs_eland/version.py` and in the `CHANGELOG.md` files, respecting the indicated formats
+. Commit the first one with "Bump version number" as commit message
```

Comment:
```suggestion
. Commit the first one with "Bump version number" as commit message.
```

Reply:
Could you move the full stop to the end of the line (after formats)? There is a gap in the contributing.md between "formats" and ".".

Reply:
Ah it is my PyCharm which does automatic line breaks on files, I should disable that on .md files